tools/generator/champions.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+import os
+from pathlib import Path
+
+from .generator import Generator
+
+
+def strip_end(text: str, suffix: str) -> str:
+    """Remove given suffix in a string"""
+    if not text.endswith(suffix):
+        return text
+    return text[:len(text) - len(suffix)]
+
+
+def make_champions(game, out_dir: Path) -> None:
+    """Generate the template to code a champion for a stechec project"""
+
+    languages = ['cxx']
+
+    for lang in languages:
+        template_dir = os.path.join('champions', lang)
+        gen = Generator(template_dir, game=game, out_dir=out_dir / lang)
+
+        for tpl in gen.env.loader.list_templates():
```

Comment:
I think I'd rather have the verbose list of files to be installed, it's more explicit and less magic.

Reply:
I have no problems with being explicit. But here the problem is that the list of files will be different for each languages. So that means having a complete list of all generated files for all generated languages. Do we want that?

Reply:
That's what I had in mind, I think we will need that anyway when we install stuff in `/usr` or need to special case some files.

Reply:
ok